packages/site/src/routes/[dictionaryId]/export/+page.svelte

```diff
@@ -24,7 +25,8 @@
   let mounted = false;
 
   onMount(async () => {
-    const entries = await getCollection<IEntry>(`dictionaries/${$dictionary.id}/words`);
+    const unconverted_entries = await getCollection<IEntry>(`dictionaries/${$dictionary.id}/words`);
+    const entries = unconverted_entries.map((entry) => convert_entry_to_current_shape(entry));
```

Comment:
@jacob-8 am I doing right? I will proceed to change the formatEntries.ts to read the converted entries correctly